Vizzuality/heco-invest
# feat: Enabling image validation

This PR adds extra validation to provided images. There is new attribute called `analyzed` which is part of image blob response and it can have three states now:
 - null --> there is no image
 - false --> there is image, but it was not analyzed yet by background job
 - true --> there is image and it was analyzed by background image
 
Links to image are returned only when `analyzed == true`.

Images which are analyzed by background job and which pretends to be images, but they are not images are automatically deleted --> from API point of view, it looks like image was not uploaded at all.

Feel free to merge this @clementprdhomme @barbara-chaves when you are ready to work on this ;)

## Testing instructions

Upload correct and spoofed image as part of Account --> I recommend to use FE here and take ID which you can use at rswag doc from console. Correct image should have `analyzed` set to `true` after couple seconds and links should be visible. Spoofed image should be automatically deleted by analyze job.

## Tracking

https://vizzuality.atlassian.net/browse/LET-1151
https://vizzuality.atlassian.net/browse/LET-1152
